homeassistant/components/tankerkoenig/base_sensor.py

```diff
@@ -0,0 +1,88 @@
+"""Base classe for tankerkoenig sensor integration."""
+import logging
+
+from homeassistant.helpers.entity import Entity
+from homeassistant.const import (
+    ATTR_ATTRIBUTION,
+    ATTR_LATITUDE,
+    ATTR_LONGITUDE,
+    STATE_CLOSED,
+    STATE_OPEN,
+)
+from .const import NAME
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_ADDRESS = "address"
+ATTR_BRAND = "brand"
+ATTR_FUEL = "fuel_type"
+ATTR_IS_OPEN = "state"
+ATTR_STATION_NAME = "station_name"
+ATTRIBUTION = "Data provided by https://creativecommons.tankerkoenig.de"
+
+ICON = "mdi:fuel"
+
+
+class FuelPriceSensorBase(Entity):
+    """Contains prices for fuels in the given station."""
+
+    def __init__(self, fuel_type, station, name=NAME):
+        """Initialize the sensor."""
+        self._data = None
+        self._station = station
+        self._station_id = station["id"]
+        self._fuel_type = fuel_type
+        self._name = name
+        self._latitude = station["lat"]
+        self._longitude = station["lng"]
+        if station["isOpen"]:
```

Comment:
`self._is_open = STATE_OPEN if station["isOpen"] else STATE_CLOSED`

Reply:
Done. I hadn't done python for a while, so I agree that my code is not very "pythonic" ;-)